app/models/miq_expression.rb

```diff
@@ -1711,12 +1711,11 @@ def self.is_plural?(field)
 
   def self.atom_error(field, operator, value)
     return false if operator == "DEFAULT" # No validation needed for style DEFAULT operator
+    return _("Ruby scripts in expressions are no longer supported.") if operator.to_s.casecmp("ruby") == 0
```

Comment:
RE: the last commit....Why change it from

```ruby
...if operator.to_s.downcase == "ruby"
```

To:

```ruby
...if operator.to_s.casecmp("ruby") == 0
```

I don't have to think about the first version.  The second one is harder to read IMHO.

Maybe drop this last commit?